packages/riipen-ui/src/components/Accordion.test.jsx

```diff
@@ -0,0 +1,150 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Accordion from "./Accordion";
+import AccordionDetails from "./AccordionDetails";
+import AccordionSummary from "./AccordionSummary";
+
+describe("<Accordion>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <AccordionDetails />;
+
+    try {
+      mount(<Accordion>{child}</Accordion>);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const summary = <AccordionSummary />;
+    const child = <AccordionDetails />;
```

Comment:
I don't think these need to be variables. I would just render them inline on lines 29-30.